### PR TITLE
Sean/deploying contract tutorial

### PR DESCRIPTION
**Pull request by smb2796**

### CODE REVIEW

File: docs/tutorials/contract-deployment.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+Before deploying a mainnet contract, it is highly recommended that you have tested liquidation and dispute bots ready to deploy. Without a network of liquidation and dispute bots, a mainnet contract is vulnerable to attacks.
+
+:::
+
+## Is your desired price identifier and collateral type supported?
+Before deploying a contract, you should verify that your desired price identifier and collateral currency is already approved on the network you are trying to deploy to.
+
+View approved price identifiers and currencies here:
+- [Approved mainnet collateral currencies](https://docs.umaproject.org/oracle/mainnet-info#current-dvm-parameters)
```
Comment (chrismaree) [inline docs/tutorials/contract-deployment.md:25]: I went and looked at this link and it shows the a list of collaters & final fees but not a simple list of collateral currencies. Is the reader simply meant to read from the list under the Final fee sections?

File: docs/tutorials/contract-deployment.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    "syntheticName" : "Synthetic Test USDETH Dec 2024", 
+    "syntheticSymbol" : "USDETH-DEC-2024", 
+    "collateralRequirement" : { "rawValue" : "1500000000000000000" }, 
+    "disputeBondPct" : { "rawValue" : "100000000000000000" }, 
+    "sponsorDisputeRewardPct" : { "rawValue" : "50000000000000000" }, 
+    "disputerDisputeRewardPct" : { "rawValue" : "200000000000000000" }, 
+    "minSponsorTokens" : { "rawValue" : "100000000000000000000" }, 
+    "withdrawalLiveness" : "7200", 
+    "liquidationLiveness" : "7200", 
+    "excessTokenBeneficiary" : "0x123456789"
```
Comment (chrismaree) [inline docs/tutorials/contract-deployment.md:68]: should we default this to the 0x address? most of th time you want this to be set to 0x0 unless you have a very spesific collateral.

File: docs/tutorials/contract-deployment.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+- `disputerDisputeRewardPct`: The percentage size of the dispute reward paid to the disputer converted to 18 decimals. Example for 20%: `{ rawValue: '200000000000000000' }`.
+- `minSponsorTokens`: The minimum number of tokens required in a sponsor position converted to 18 decimals. Example for 100 tokens: `{ rawValue: '100000000000000000000' }`.
+- `withdrawalLiveness`: The length of the [withdrawal liveness period](https://docs.umaproject.org/synthetic-tokens/glossary#withdrawal-liveness-period) in seconds. Example for 2 hours: `7200`.
+- `liquidationLiveness`: The length of the [liquidation liveness period](https://docs.umaproject.org/synthetic-tokens/glossary#liquidation-liveness-period) in seconds. Example for 2 hours: `7200`.
+- `excessTokenBeneficiary`: The Ethereum address that will receive any sort of excess collateral accrued by interest bearing collateral currencies, such as rDAI.
+
+This is what the parameters object will look like when correctly formatted.
+
+```json
+{
```
Comment (chrismaree) [inline docs/tutorials/contract-deployment.md:55]: does this need to be wrapped into a ` punctuation mark or something for etherscan support?

File: docs/tutorials/contract-deployment.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+To add a new price identifier or collateral currency to mainnet, please propose one to UMA voters by following the [UMIP process](https://docs.umaproject.org/governance/umips).
+
+To add a new price identifier or collateral currency to Kovan, please follow these [instructions](https://docs.umaproject.org/governance/adding-price-id#adding-a-price-identifier-to-the-kovan-testnet) or contact the UMA core team in Discord.
+
+## EMP parameters
+
+The mandatory EMP parameters are:
+
+- `expirationTimestamp`: The UNIX timestamp that the EMP will expire at. Example December 31, 2024 expiry: `1735682400`.
```
Comment (chrismaree) [inline docs/tutorials/contract-deployment.md:38]: perhaps link to https://www.unixtimestamp.com/ to help with finding the associated timestamp for a given date.

**Review by chrismaree — APPROVED** [review]
THis looks great! I have a few comments/nits but other than them this is awesome :)

File: docs/tutorials/contract-deployment.md (hunk 3 above)
Comment (smb2796) [reply docs/tutorials/contract-deployment.md:55]: Nope - it's just the object with no wrapping punctuation

File: docs/tutorials/contract-deployment.md (hunk 1 above)
Comment (smb2796) [reply docs/tutorials/contract-deployment.md:25]: Yes, I will specify!

File: docs/tutorials/contract-deployment.md (hunk 4 above)
Comment (smb2796) [reply docs/tutorials/contract-deployment.md:38]: Good idea!

**Pull request merged**